_dev/.stylelintrc

```diff
@@ -1,12 +1,12 @@
 {
   "ignoreFiles": [
-    "../packages/fxa-content-server/**"
+    "../packages/**/tailwind.out.scss"
```

Comment:
Subtle but big change, don't be fooled by the svelte diff.
fxa-content-server is no longer being bulk ignored. 👀